corehq/apps/hqcase/templates/hqcase/xml/case_block.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' ?>
 <system version="1" uiVersion="1" xmlns="{{ xmlns }}" xmlns:orx="http://openrosa.org/jr/xforms">
     <orx:meta xmlns:cc="http://commcarehq.org/xforms">
-        {% if not device_id %}<orx:deviceID />{% else %}<orx:deviceID>{{ device_id }}</orx:deviceID>{% endif %}
+        <orx:deviceID>{{ device_id }}</orx:deviceID>
```

Comment:
@dannyroberts I don't think there's any semantic difference between `<element></element>` and `<element />` in XML, so making this like all the rest. Maybe you had a different reason for doing this one differently?

Reply:
Nope, I'm cool with this. I was probably just very literally going off an example that had `<orx:deviceID />` in it, and didn't think harder about it. Agree this is more straightforward, and just as good.